### ISSUE TITLE
[Feature request] Clear cache on exit

### ISSUE THREAD

**Issue by RickyM7** [TwidereProject/Twidere-Android#1384]
This is basically it, when this option is activated, when closing the app the cache is automatically cleared.
I think this would be very useful, especially for phones with little storage.

**Comment by jk234ert** [comment]
I agreed this is useful to keep users' devices clean.
While we already have a "Clear cache" button, a "Clear cache on exit" option would be a lower priority feature in the new version

**Comment by RickyM7** [comment]
@jk234ert Yes, this is something useful, but it is definitely not urgent, it is something to be seen later.
